### PR TITLE
Feature/remove unapplicable guides

### PR DESCRIPTION
**Pull request by gaganawhad**
Remove guides that are not applicable to us

### CODE REVIEW

**Comment by gaganawhad** [comment]
@benhutton FYI! Not sure if you get notifications for being assigned to this PR so I decided to write a comment.

File: style/README.md [hunk 1]
```diff
@@ -1,84 +0,0 @@
-Style
```
Comment (benhutton) [inline style/README.md]: @gaganawhad maybe this file should stay and provide a table of contents to the rest of the `/styles` directory?

**Comment by benhutton** [comment]
@gaganawhad there are lines in the root README.md file that link to some of this deleted content

File: style/README.md (hunk 1 above)
Comment (gaganawhad) [inline style/README.md]: @benhutton you are right. I'll make updates to this file instead of delete it.

**Comment by gaganawhad** [comment]
@benhutton hmmm.... how about we wait on making updates to the root README? There are quite a few updates and additions that would be required there, so may be we can update it once we feel setup with our guides?

**Comment by benhutton** [comment]
@gaganawhad works for me

**Comment by gaganawhad** [comment]
@benhutton There you go! I added the README in with the appropriate deletions. As you have very likely guessed I am doing the work in this PR with a broad brush. I want to then go and make specific deletions / updates to what remains after we merge this in.

**Pull request merged**